src/Login.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {withRouter, Link} from 'react-router-dom'
+
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      password: '',
+      error: '',
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleInputChange(event) {
+    const target = event.target;
+    const value = target.value;
+    const name = target.name;
+
+    this.setState({
+      [name]: value
+    });
+  }
+
+  handleSubmit(event) {
```

Comment:
Add an if statement for to check inputs from user. This could call a new method "checkInputs" before submitting to ensure that user has filled it out properly. For basis, look at checkPasswords() method from Register.js